src/rust/ensogl/src/display/shape/primitive/def/var.rs

```diff
@@ -473,6 +474,20 @@ where T: Acos<Output=T> {
     }
 }
 
+impl<T> Atan2 for Var<T>
+    where T: Atan2<Output=T> + RefInto<Glsl> + Copy{
+    type Output = Var<T>;
+    fn atan2(&self, other:Self) -> Self {
+        match (self, &other) {
+            (Var::Static(lhs), Var::Static(rhs)) => Var::Static(lhs.atan2(*rhs)),
+            _ => {
+                    let code = format!("atan({},{})",self.glsl(),other.glsl());
+                    Var::Dynamic(code.into())
+            }
```

Comment:
make it single line pls (without `=>` alignment). Its not more clear the way its now.